db/migrate/20241120024939_create_evaluations.rb

```diff
@@ -0,0 +1,18 @@
+class CreateEvaluations < ActiveRecord::Migration[7.2]
+  def change
+    create_table :evaluations do |t|
+      t.references :user, null: false, foreign_key: true
+      t.references :evaluation_form, null: false, foreign_key: true 
+      t.references :submission, null: false, foreign_key: true
+
+      t.text :additional_comments
+      t.text :revision_comments
+      t.integer :status, default: 0, null: false
+      t.integer :total_score, default: nil
+
+      t.timestamps
```

Comment:
`EvaluatorSubmissionAssignment` keeps the assignment `recused` status along with assignment status.
I think we can track the not_started/in_progress/completed status a bit differently on evaluations...
1. The status is considered `not_started` if the `Evaluation` record does not exist for that EvaluatorSubmissionAssignment. The Evaluation will not be created yet when an evaluator user is assigned to the submission, we only create the `EvaluatorSubmissionAssignment` record.
2. The status is considered `in_progress` if the `Evaluation` record exists but is not completed. We will create the Evaluation record when the evaluator starts evaluating and saves their scores as a draft or completes their scoring.
3. The status is considered `completed` if the `Evaluation` record's `completed_at` timestamp is not null. The `completed_at` timestamp is set when the user clicks "Complete Evaluation" and then "Yes" on the "Complete your evaluation?" confirmation modal.

```suggestion
      t.references :user, null: false, foreign_key: true
      t.references :evaluation_form, null: false, foreign_key: true 
      t.references :submission, null: false, foreign_key: true
      t.references :evaluator_submission_assignment, null: false, foreign_key: true

      t.text :additional_comments
      t.text :revision_comments
      t.integer :total_score, default: nil

      t.datetime :completed_at
      t.timestamps
```